process/coordinator/process.go

```diff
@@ -400,6 +400,10 @@ func (tc *transactionCoordinator) ProcessBlockTransaction(
 		return process.ErrNilBlockBody
 	}
 
+	if tc.isMaxBlockSizeReached(body) {
+		return process.ErrMaxBlockSizeReached
```

Comment:
so this will prevent the validators sign upon a wrongly constructed block that contains too many txhashes?

Reply:
Exactly